SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -79,6 +81,9 @@ main() {
         s3_put_no_index $extracted_file $path/$file_basename
     done
 
+    echo "" | notify_by_email $recipient "Successfully published SOOP_CO2 file '$path' "
+    echo "" | notify_by_email $BACKUP_RECIPIENT "Successfully published SOOP_CO2 file '$path'"
+
```

Comment:
(General comment) Sending two emails to the uploader for each file (+2 to yourself) is ok if only a few files are uploaded at a time. In the long run it would be much better to have a way to generate a single report summarising all relevant events during the incoming handling, and just send it once at the end, whether the process terminated happily or with an error.